ntttcp/ntttcp.Config.json

```diff
@@ -49,7 +49,7 @@
             "Options"       : ""
         },
         "udp" : {
-            "BufferLen"     : [512, 1372, 1472],
+            "BufferLen"     : [512, 1000, 1372, 1400, 1472],

```

Comment:
When you visualize these results with PerfViz, you will need to use multi-pivot.  IIRC, multi-pivot was not in use in prior measurements.  But I may be mistaken.